build.gradle

```diff
@@ -25,15 +25,15 @@ subprojects {
 
   ext {
     versions = [
-      opentelemetrySdk           : "1.23.0",
+      opentelemetrySdk           : "1.24.0",
+      opentelemetryJavaagent     : "1.25.0-SNAPSHOT",
```

Comment:
Shouldn't we use 1.24.0? This looks like it's built on the snapshot of 1.25.0 of the SDK